src/index.js

```diff
@@ -98,6 +98,10 @@ export default class SauceLabs {
             return ::this._uploadJobAssets
         }
 
+        if (propName === 'createTestResult') {
+            return ::this._createTestResult
+        }
```

Comment:
This is not needed. The lib can figure this out by consuming the open api spec. So it should work the same way without this function.

Reply:
updating.

Reply:
I tried with the way that you mentioned. This `createResultJob` can't run successfully because of lack of `this._api`. It seems that a lot of functions are not bounden using this way.